source/guide_rust_server.rst

```diff
@@ -0,0 +1,111 @@
+.. author:: tpraxl <kontakt@macrominds.de>
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/rust-logo-blk.svg
+      :align: center
+
+###########
+Rust Server
+###########
+
+It couldn't be simpler to run a Rust web server on Uberspace.
+
+Install Cargo, checkout your server code, install the server as a supervisord_ service and start it. Done.
+
+----
+
+.. note:: For this guide you might want to be familiar with the basic concepts of
+
+  * Rust_
+  * htaccess_
+  * supervisord_
+
+Prerequisites
+=============
+
+We need to install Rust_ first:
+
+::
+
+ [rust@taylor ~]$ curl https://sh.rustup.rs -sSf | sh
+ [rust@taylor ~]$ # Choose 1) for the default installation option that also sets up your PATH
+ [rust@taylor ~]$ source ~/.cargo/env
+
+Providing a Rust Server
+=======================
+
+To simplify this recipe, just follow the thruster-intro-tutorial_ to create your Hello World Server.
+
+Connect it to the outside world
+===============================
+
+.. warning:: Replace ``<ubername>`` with your Uberspace name!
+
+::
+
+  [rust@taylor ~]$ vim /var/www/virtual/<ubername>/html/.htaccess
+
+.. warning:: Replace ``4321`` with the port of your server (``4321`` is the port used in the thruster-intro-tutorial)
+
+.. code-block:: apacheconf
+
+  RewriteEngine On
+  # Using this rule, everything under <ubername>.uber.space/rust will be mapped to your Rust server
```

Comment:
Style guide:
> If any applications need to bind to one or more ports (like Node.js apps), use port 9000 - 9010 in your examples, which are never free on our servers.

I assume you didn't read the style guide and I stop reviewing here. Please update your guide according to the style guide and request another review.